instrumentation/tomcat/tomcat-7.0/javaagent/src/test/groovy/io/opentelemetry/javaagent/instrumentation/tomcat/v7_0/TomcatAsyncTest.groovy

```diff
@@ -130,4 +130,11 @@ class TomcatAsyncTest extends HttpServerTest<Tomcat> implements AgentTestTrait {
         break
     }
   }
+
+  @Override
+  Set<AttributeKey<?>> httpAttributes(ServerEndpoint endpoint) {
```

Comment:
What about the response headers? And don't we have a shared consts file where we can save the "http.request.headers" (and similar consts)?

Reply:
The instru I've added is only for the request. I can add to the response as well (the request is more common use isn't it?)

Regarding the constants - so there is SEMATNIC_ATTRIBUTES for java otel but - they are in a different repo (and there is no attribute for the http headers) - so if we want we need to contribute to the other repo and afterwards use it

Reply:
1. Let's collect the response headers as well
2. Why not create a class of our own that extends it? This is what we do in other languages